### ISSUE TITLE
Does vllm support model.foward()

### ISSUE THREAD

**Issue by cobraheleah** [vllm-project/vllm#713]
I want to test my model in MMLU(Massive Multitask Language Understanding)task.So i need to get the last token logits by model.foward()，does vllm support model.foward() ?

**Comment by LSC527** [comment]
You can set `max_tokens=1` and `logprobs=vocab_size` in `SamplingParams` to get full logits of the last token.

**Comment by RaymondHQR** [comment]
> You can set `max_tokens=1` and `logprobs=vocab_size` in `SamplingParams` to get full logits of the last token.

Hi, I wonder if you can provide a detailed code of this, pls help me, thx :)
